multimedia/winff/winff.SlackBuild

```diff
@@ -70,12 +69,12 @@ set -e
 trap 'echo "$0 FAILED at line $LINENO!" | tee -a  $OUTPUT/error-${PRGNAM}.log' ERR
 
 rm -rf $PKG
+rm -rf $TMP
```

Comment:
you can't rm $TMP as it will break the machine

Reply:
rm $PKG already delete the directory created by extracting the source, no need to delete the whole $TMP